code/modules/reagents/chemistry/recipes/toxins.dm

```diff
@@ -6,6 +6,12 @@
 	required_reagents = list("ethanol" = 1, "oxygen" = 1, "silver" = 1)
 	required_temp = 420
 
+/datum/chemical_reaction/shellblood
+	name = "shell blood"
+	id = "shell_blood"
+	results = list("shellblood" = 10)
+	required_reagents = list("sacid" = 1, "iron" = 2, "radium" = 1, "water" = 1)
```

Comment:
It does, it's gonna be necessary if you have to replace a shell's blood. It's not the first reaction to do so either.